ui/app/AppLayouts/Communities/popups/FinaliseOwnershipPopup.qml

```diff
@@ -0,0 +1,335 @@
+    padding: 0
+
+    component CustomText : StatusBaseText {
+        Layout.fillWidth: true
+
+        wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+        lineHeight: 1.2
+    }
+
+    contentItem: StatusScrollView {
+        id: scrollView
+
+        contentWidth: availableWidth
+        contentHeight: loader.item.height
+
+        padding: Style.current.padding
+
+        Loader {
+            id: loader
+
+            width: scrollView.availableWidth
+            sourceComponent: initialPanel
+            state: d.init
+            states: [
+                State {
+                    name: d.init
+
+                    PropertyChanges { target: loader; sourceComponent: initialPanel }
+                    PropertyChanges {
+                        target: acceptBtn
+                        text: qsTr("Finalise %1 ownership").arg(root.communityName)
+                        enabled: true
+
+                        onClicked: loader.state = d.finalise
+                    }
+                    PropertyChanges {
+                        target: rejectBtn
+                        visible: true
+
+                        onClicked: root.rejectClicked()
+                    }
+                    PropertyChanges { target: backButton; visible: false }
+                },
+                State {
+                    name: d.finalise
+
+                    PropertyChanges { target: loader; sourceComponent: finalisePanel }
+                    PropertyChanges {
+                        target: acceptBtn
+                        text: qsTr("Make this device the control node and update smart contract")
+                        enabled: d.ackCheck && !root.isFeeLoading && root.feeErrorText === ""
+
+                        onClicked: {
+                            root.finaliseOwnershipClicked()
+                            close()
+                        }
+                    }
+                    PropertyChanges { target: rejectBtn; visible: false }
+                    PropertyChanges {
+                        target: backButton
+                        visible: true
+
+                        onClicked: loader.state = d.init
+                    }
+                }
+            ]
+        }
+    }
+
+    header: StatusDialogHeader {
+        headline.title: qsTr("Finalise %1 ownership").arg(root.communityName)
+        actions.closeButton.onClicked: root.close()
+        leftComponent: StatusSmartIdenticon {
+            asset.name: root.communityLogo
+            asset.isImage: !!asset.name
+        }
+    }
+
+    footer: StatusDialogFooter {
+        spacing: Style.current.padding
+        rightButtons: ObjectModel {
+            StatusFlatButton {
+                id: rejectBtn
+
+                text: qsTr("I don't want to be the owner")
+
+                onClicked: {
+                    root.rejectClicked()
+                    close()
+                }
+            }
+
+            StatusButton {
+                id: acceptBtn
+
```

Comment:
I would add `Layout.fillWidth: implicitWidth > parent.width` here to elide properly when translation is wider than available space.